spec/features/user_views_prs_spec.rb

```diff
@@ -116,9 +116,9 @@
       click_on "rails"
     end
 
-    expect(page).to have_content("An Ember PR")
-    expect(page).to have_content("A Rails PR")
-    expect(page).to have_content("A long pr")
+    expect(page).to have_content("An Ember PR", count: 1)
+    expect(page).to have_content("A Rails PR", count: 1)
+    expect(page).to have_content("A long pr", count: 1)
```

Comment:
I'm not sure if just modifying this existing test would suffice...if you prefer a separate one, let me know.